ivy/functional/frontends/torch/nn/functional/pooling_functions.py

```diff
@@ -191,3 +191,30 @@ def lp_pool1d(input, norm_type, kernel_size, stride=None, ceil_mode=False):
         )
 
     return ivy.pow(ivy.multiply(out, kernel_size), ivy.divide(1.0, norm_type))
+
+
+@to_ivy_arrays_and_back
+def lp_pool2d(input, norm_type, kernel_size, stride=None, ceil_mode=False):
```

Comment:
`ceil_mode` seems to be ignored throughout the implementation, I noticed that `ivy.avg_pool2d` does not include that parameter, while `ivy.max_pool2d` does, we should be supporting `ceil_mode` in `ivy.avg_pool2d` as well.

Reply:
As of now `ceil_mode` is not handled in the ivy api though, so for this specific function it should ok to leave like this for the time being right?

Reply:
Actually, yes we can ignore it for now.